app/controllers/api/v0/swagger/models/highlight.rb

```diff
@@ -42,25 +42,69 @@ module Api::V0::Swagger::Models::Highlight
     end
   end
 
+  swagger_schema :GetHighlights do
+    key :required, [:source_type]
+    property :page do
+      key :type, :integer
+      key :description, 'The page number'
```

Comment:
Need to be more specific here somehow so people don't think "OpenStax page number".  E.g. "The page number of paginated results, zero-indexed".  

Specify the default? (guessing it is zero).